pkg/omf/cli/omf.destroy.fish

```diff
@@ -1,10 +1,14 @@
 function omf.destroy -d "Remove Oh My Fish"
   echo (omf::dim)"Removing Oh My Fish..."(omf::off)
 
-  omf.remove_package (basename $OMF_PATH/pkg/*) >/dev/null ^&1
+  for pkg in (basename $OMF_PATH/pkg/*)
+    omf.remove_package $pkg >/dev/null ^&1
+  end
 
-  if test -e "$HOME/.config/fish/config.copy"
-    mv "$HOME/.config/fish/config".{copy,fish}
+  set -l localbackup (find $HOME/.config/fish/ -regextype posix-extended -regex '^.*fish/config\.[[:digit:]]+\.copy$' |\
+    sort -r |head -1)
+  if test -n $localbackup
+    mv $localbackup "$HOME/.config/fish/config.fish"
```

Comment:
You could use XDG specification (`$XDG_CONFIG_HOME`). Using a variable to store it and fallback to `~/.config` when empty would be greater.